### DISCUSSION TITLE
Update API to return list of specified posts

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#246] [Medium CIPs]
Let's update `get-posts-stateless` API, so we can pass array of post-hashes as parameter, so we get the list of posts objects in response.

Update needs to be done here  https://github.com/deso-protocol/backend/blob/aff732a2ad05d88cf634b678d15393bd93fb97d5/routes/post.go#L20

@diamondhands0 @lazynina @bluepartyhat 

PS sure it must support pagination, similar like when fetching all posts

**Comment by matt-mcmarsh** [comment]
sounds good for collections

**Comment by lazynina** [comment]
I suggest a new endpoint instead. This endpoint is already overloaded for getting feeds. Something like GetPostsByPostHashHexes and returns a map of post to post hash hex

**Reply by brootle** [reply]
@lazynina  for me anything would work, just need the ability to fetch posts by sending array of post hashes 😀

**Reply by erwin-willems** [reply]
This would help us (DeSocialWorld) also a lot, because we have a lot of custom feeds. We currently have built this ourselves in an API that just fetches the single posts from an array, but running this in the backend could improve performance.
